src/Component/muro.js

```diff
@@ -0,0 +1,126 @@
+import { onNavigate } from '../main.js';
+import { cerrar } from '../lib/index.js';
+import { user } from '../lib/index.js';
```

Comment:
Consistencia si todo en ingles, todo ingles
```suggestion
import { onNavigate } from '../main.js';
import { signOut } from '../lib/index.js';
import { user } from '../lib/index.js';
```